modules/ROOT/pages/apikit-4-xml-reference.adoc

```diff
@@ -1,91 +1,97 @@
+|`keepRamlBaseUri` |N |Alias for the equivalent `keepApiBaseUri` attribute.
 
-This attribute is deprecated since APIkit 1.2.0 and only kept for compatibility with APIkit 1.1.x. +
-It's recommended that API implementations use "keepApiBaseUri" instead of “keepRamlBaseUri”.
-|*name* |Y |Configuration name that other elements, such as apikit:router, reference
-|*raml* |Y | An alias for the equivalent "api" attribute.
+This attribute is deprecated since APIkit 1.2.0. Included to provide compatibility with APIkit 1.1.x. +
+Use `keepApiBaseUri` instead of `keepRamlBaseUri` in your API implementations.
+|name |Y |Configuration name that other elements, such as `apikit:router`, reference.
+|raml |Y |Alias for the equivalent `api` attribute.
 
-This attribute is deprecated since APIkit 1.2.0 and only kept for compatibility with APIkit 1.1.x. +
-It's recommended that API implementations use "api" instead of “raml”.
-|*parser* | N | Specifies the parser used to validate requests.
+This attribute is deprecated since APIkit 1.2.0. Included to provide compatibility with APIkit 1.1.x. +
+Use `api` instead of `raml` in your API implementations.
+|parser | N |Parser used to validate requests.
 
-*This attribute is valid for APIkit 1.2.0 and later*
+Attribute valid for APIkit 1.2.0 and later.
 
 Possible values are:
 
-* AUTO: The APIkit Router will try to initiate using AMF, it's not possible (it has errors), then it will try with the RAML Parser.
-* AMF: The APIkit Router will use AMF to read the API and validate incoming requests.
-* RAML: The APIkit Router will use RAML Parser to read the API and validate incoming requests. This is similar to previous APIkit versions.
+* AUTO
++
+*APIkit Router* initiates by using AMF. If it has errors, it initiates by using the RAML parser.
+* AMF
++
+*APIkit Router* uses AMF to read the API and validate incoming requests.
+* RAML
++
+*APIkit Router* uses RAML parser to read the API and validate incoming requests. Similar to previous APIkit versions.
 
-*Default Value: AUTO*.
+Default Value: AUTO.
 
-| *api* | N | Root API file location.
+| api | N | Root API file location.
 
-*This attribute is valid for APIkit 1.2.0 and later*
+Attribute valid for APIkit 1.2.0 and later.
 
-It can be any of a URL, the relative path to a file located under `src/main/resources/api`, or the API Sync resource path.
+It can be any URL, the relative path to a file located under `src/main/resources/api`, or the `API Sync` resource path.
 
-| *keepApiBaseUri* | N | *This attribute is valid for APIkit 1.2.0 and later*
-
-When set to *false*, it rewrites the `baseUri` in the RAML definition of the API. +
-If set to *true* it does not rewrite the `baseUri` field.
+| `keepApiBaseUri` | N | Attribute valid for APIkit 1.2.0 and later.
 
+When set to `false`, it rewrites the `baseUri` in the RAML definition of the API. +
+If set to `true`, it does not rewrite the `baseUri` field.
```

Comment:
comment on 82

Reply:
It referes to APIkit. I'll add it. Thanks!